src/Qowaiv.DomainModel.TestTools/AggregateRootAssert.cs

```diff
@@ -22,7 +22,7 @@ public static class AggregateRootAssert
     /// </param>
     [Assertion]
     public static void HasUncommittedEvents<TAggregate, TId>(Result<TAggregate> actualAggregate, params object[] expectedEvents)
-        where TAggregate : AggregateRoot<TAggregate, TId>, new()
+        where TAggregate : Aggregate<TAggregate, TId>, new()
```

Comment:
The sb.AppendLine below still mentions aggregate root.

Reply:
Will be dropped with #37